interactions/client.py

```diff
@@ -378,10 +380,13 @@ def decorator(coro: Coroutine) -> Callable[..., Any]:
                     )
                 _option: Option
                 for _option in options:
-                    if not _option._json.get("description"):
+                    if not _option._json.get("description") and _option not in (
+                        OptionType.SUB_COMMAND,
+                        OptionType.SUB_COMMAND_GROUP,
+                    ):
```

Comment:
shouldn't this be `_option._json.get("type")` or am I thinking too complicated?

Reply:
Oh oops. I pressed undo too many times. Will fix